MyCompany.Seeq.Link.Connector.MyConnector.Test/MyCompany.Seeq.Link.Connector.MyConnector.Test.csproj

```diff
@@ -77,11 +77,11 @@
     <Reference Include="RestSharp, Version=106.12.0.0, Culture=neutral, PublicKeyToken=598062e77f915f75, processorArchitecture=MSIL">
       <HintPath>..\packages\RestSharp.106.12.0\lib\net452\RestSharp.dll</HintPath>
     </Reference>
-    <Reference Include="Seeq.Link.SDK, Version=64.0.3.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Seeq.Link.SDK.64.0.3\lib\net48\Seeq.Link.SDK.dll</HintPath>
+    <Reference Include="Seeq.Link.SDK, Version=65.0.0-dev.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
65.0.0-dev is a release created from [this branch](https://github.com/seeq12/crab/pull/6517).
I manually uploaded all nupkg files to NuGet as 65.0.0-dev to avoid creating already a 65.0.0 (in Nuget there is no way to delete packages and a try to deliver 65.0.0 again will fail - therefore I used this `-dev` suffix)

Reply:
Yes, I think it would be beneficial. We don't need this very often but I think it is beneficial to have it. Can you log a CRAB?